docs: add a reference to template-sync tool (#1849)
api-platform/docs: symfony/index.md

Of course, you can also use your favorite HTTP client to query the API.
We are fond of [Hoppscotch](https://hoppscotch.com), a free and open source API client with good support of API Platform.

> ## Update Your Project

Reviewer: WDYT?
```suggestion
## Keep Your Project in Sync with the API Platform Template
```

Author: It looks more specific

You have started a project with the API Platform template and you would like to benefit from the latest enhancements introduced since you created your project (i.e. [FrankenPHP](https://frankenphp.dev/). Juste use this git based tool
[The *template-sync* project](https://github.com/coopTilleuls/template-sync) got you covered.

Run the following command to import the changes since your last update:

```console

> curl -sSL https://raw.githubusercontent.com/mano-lis/template-sync/main/template-sync.sh | sh -s -- https://github.com/api-platform/api-platform

Reviewer: IMHO

```suggestion
curl -sSL https://raw.githubusercontent.com/coopTilleuls/template-sync/main/template-sync.sh| sh -s -- https://github.com/api-platform/api-platform
```

```

> Resolve potential conflicts, run `git cherry-pick --continue` and you are done!

Reviewer: ```suggestion

Resolve potential conflicts, run `git cherry-pick --continue` and you are done!
```

> [Full documentation](https://github.com/coopTilleuls/template-sync/blob/main/README.md)

Reviewer: ```suggestion
For more details, refer to the [coopTilleuls/template-sync documentation](https://github.com/coopTilleuls/template-sync/blob/main/README.md)
```
